host/base/host.DockerFIle

```diff
@@ -4,9 +4,11 @@ ARG JDK_NAME=jdk8u362-b09
 ARG JAVA_HOME=/usr/lib/jvm/adoptium-8-x64
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS runtime-image
 ARG HOST_VERSION
+ARG EXTENSION_BUNDLE_VERSION
 ARG JAVA_VERSION
 ARG JDK_NAME
 ARG JAVA_HOME
+ARG EXTENSION_BUNDLE_CDN_URL=https://functionscdn.azureedge.net/public
```

Comment:
Do we need to target only public/released extension bundle?

Reply:
no, we can create image from custom bundle as well, in that case need to set the CDN url build arguments
check the PR description.